components/explorer/src/lib.rs

```diff
@@ -354,29 +371,57 @@ impl<'a> IntoIterator for &'a BlockWithTransactions {
 /// { "type": "success" }
 /// ```
 ///
-/// For transactions that return an [`ExecutionError`], `status` contains the error code
-/// and an optional description, i.e., has the following type in the
-/// [`Flow`] / [`TypeScript`] notation:
-///
-/// ```javascript
-/// { type: 'service_error', code: number, description?: string }
-/// ```
-///
-/// For transactions that have resulted in a panic, `status` contains an optional description
-/// as well:
-///
-/// ```javascript
-/// { type: 'panic', description?: string }
+/// For transactions that cause an [`ExecutionError`], `status` contains the error code
+/// and an optional description, i.e., has the following type in the [TypeScript] notation:
+///
+/// ```typescript
+/// type Error = {
+///   type: 'service_error' | 'dispatcher_error' | 'runtime_error' | 'unexpected_error',
+///   code?: number,
+///   description?: string,
+///   runtime_id: number,
+///   call_site?: CallSite,
+/// };
+///
+/// type CallSite = MethodCallSite | HookCallSite;
+///
+/// type MethodCallSite = {
+///   call_type: 'method',
+///   instance_id: number,
+///   interface?: string,
+///   method_id: number,
+/// };
+///
+/// type HookCallSite = {
+///   call_type: 'constructor' | 'before_transactions' | 'after_transactions',
+///   instance_id: number,
+/// };
 /// ```
 ///
-/// [`Transaction`]: ../blockchain/trait.Transaction.html
-/// [`TxLocation`]: ../blockchain/struct.TxLocation.html
-/// [`ListProof`]: ../../exonum_merkledb/indexes/proof_list/struct.ListProof.html
-/// [`Hash`]: ../../exonum_crypto/struct.Hash.html
-/// [`ExecutionStatus`]: ../runtime/error/struct.ExecutionStatus.html
-/// [`ExecutionError`]: ../runtime/error/struct.ExecutionError.html
-/// [`Flow`]: https://flow.org/
-/// [`TypeScript`]: https://www.typescriptlang.org/
+/// Some explanations:
```

Comment:
Nit: "Some explanations" looks a bit awkward. I'd either shorten it to just "Explanations" or expand to show what is explained in details.